DataFormats/Headers/include/Headers/TimeStamp.h

```diff
@@ -28,7 +28,7 @@ namespace header {
 // https://lhc-machine-outreach.web.cern.ch/lhc-machine-outreach/collisions.htm
 // https://www.lhc-closer.es/taking_a_closer_look_at_lhc/0.buckets_and_bunches
 
-namespace LHCClockParameter {
+namespace lhc_clock_parameter {
```

Comment:
```suggestion
namespace lhc_clock_parameter
{
```

Reply:
@matthiasrichter : I would honestly separate fixing the namespaces from fixing formatting. Will merge this now. (In any case this file seems to old and deprecated).